config/routes.rb

```diff
@@ -1,60 +1,7 @@
 Rails.application.routes.draw do
   root 'static_pages#home'
   get 'guide' => 'static_pages#guide'
+  get 'contact' => 'static_pages#contact'
```

Comment:
Nit: I'm a complete nerd about this kinda crap.  Please line up the RHS and LHS of the hashrocket.

`Nit` indicates the suggested delta following is at your discretion ;)